code/modules/mob/living/basic/lavaland/goliath/tentacle.dm

```diff
@@ -41,16 +47,23 @@
 /// Grab everyone we share space with. If it's nobody, go home.
 /obj/effect/goliath_tentacle/proc/grab()
 	for (var/mob/living/victim in loc)
-		if (victim.stat == DEAD || HAS_TRAIT(victim, TRAIT_TENTACLE_IMMUNE))
+		if (victim.stat == DEAD || HAS_TRAIT(victim, TRAIT_TENTACLE_IMMUNE) || caster?.faction_check_mob(victim))
 			continue
 		balloon_alert(victim, "grabbed")
 		visible_message(span_danger("[src] grabs hold of [victim]!"))
-		victim.adjustBruteLoss(rand(min_damage, max_damage))
+		var/damage_dealt = rand(min_damage, max_damage)
+		victim.adjustBruteLoss(damage_dealt)
+		if(caster)
+			log_combat(caster, victim, "grabbed with a ", src, trophy_spawned ? "crusher-spawned" : "")
+		if(trophy_spawned)
+			SEND_SIGNAL(src, COMSIG_CRUSHER_SPELL_HIT, victim, caster, damage_dealt)
```

Comment:
Hmm, we are sending this signal from a lot of places, which is against our usual design for signals.
I understand _why_ we are though and it would be annoying to have a new signal for every one of these. Tricky.